src/Component/ParserFeedback/ParserFeedback.tsx

```diff
@@ -0,0 +1,130 @@
+/* Released under the BSD 2-Clause License
+ *
+ * Copyright © 2022-present, terrestris GmbH & Co. KG and GeoStyler contributors
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * * Redistributions of source code must retain the above copyright notice,
+ *   this list of conditions and the following disclaimer.
+ *
+ * * Redistributions in binary form must reproduce the above copyright notice,
+ *   this list of conditions and the following disclaimer in the documentation
+ *   and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { Alert } from 'antd';
+import { ReadStyleResult, WriteStyleResult } from 'geostyler-style';
+import React from 'react';
+
+export interface ParserFeedbackProps {
+  feedback: ReadStyleResult | WriteStyleResult;
+}
+
+/**
+ * Checkbox field for a boolean filter value.
+ */
+export const ParserFeedback: React.FC<ParserFeedbackProps> = ({
+  feedback
+}) => {
+
+  if (!feedback || Object.keys(feedback).length < 1) {
+    return null;
+  }
+
+  const {
+    errors,
+    warnings,
+    unsupportedProperties
+  } = feedback;
+
+  const warningAlerts = warnings?.map((warning, index) => (
+    <Alert
+      showIcon
+      key={`warning-${index}`}
+      type='warning'
+      message={warning}
+    />
+  ));
+
+  const errorAlerts = errors?.map((error, index) => (
+    <Alert
+      showIcon
+      key={`error-${index}`}
+      type='error'
+      message={error.message}
+    />
+  ));
+
+  const strings: string[] = [];
+
+  const getTextualRepresentation = (value: any) => {
+    let text;
+    if (typeof value === 'string' || value instanceof String) {
+      if (value === 'none') {
+        text = 'is not supported.';
```

Comment:
Shouldn't these texts be part of i18n?

Also I think it might lead to less confusion by changing the texts to

```
is not supported by used parser.
```

or something similar.